lib/emarsys/client.rb

```diff
@@ -12,25 +12,25 @@ def password
     end
 
     def x_wsse_string
-      string = 'UsernameToken '
-      string += 'Username="' + username + '", '
-      string += 'PasswordDigest="' + header_password_digest + '", '
-      string += 'Nonce="' + header_nonce + '", '
-      string += 'Created="' + header_created + '"'
-      string
+      str = <<-STRING
+      UsernameToken Username="#{username}", PasswordDigest="#{header_password_digest}", Nonce="#{header_nonce}", Created="#{header_created}"
+      STRING
+      str.strip
```

Comment:
I absolutely agree. However, let's focus on the country feature in this PR. Could you add it to a separate PR please?